misc/python/materialize/cli/run.py

```diff
@@ -506,12 +512,8 @@ def _macos_codesign(path: str) -> None:
     command = ["codesign"]
     command.extend(["-s", "-", "-f", "--entitlements"])
 
-    # write our entitlements file to a temp path
-    temp = tempfile.NamedTemporaryFile()
-    temp.write(bytes(MACOS_ENTITLEMENTS_DATA, "utf-8"))
-    temp.flush()
-
-    command.append(temp.name)
+    macos_entitlements_data = MZ_ROOT / "misc" / "dist" / "macos-entitlements-data.xml"
+    command.append(macos_entitlements_data)
```

Comment:
Doesn't need to be shared anymore, but still feels better than creating a temporary file on every run.

Reply:
Counterpoint: tempfiles are cheap, and the old approach kept the entitlement data nicely encapsulated inside of this script. I think it's kind of a wash, so fine to leave this in.